packages/utils/src/shorthand.ts

```diff
@@ -0,0 +1,519 @@
+    'background-attachment',
+    'background-clip',
+    'background-color',
+    'background-image',
+    'background-origin',
+    'background-position',
+    'background-repeat',
+    'background-size',
+  ],
+  border: [
+    'border-block',
+    'border-block-end',
+    'border-block-end-color',
+    'border-block-end-style',
+    'border-block-end-width',
+    'border-block-start',
+    'border-block-start-color',
+    'border-block-start-style',
+    'border-block-start-width',
+    'border-bottom',
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+    'border-inline',
+    'border-inline-end',
+    'border-inline-end-color',
+    'border-inline-end-style',
+    'border-inline-end-width',
+    'border-inline-start',
+    'border-inline-start-color',
+    'border-inline-start-style',
+    'border-inline-start-width',
+    'border-left',
+    'border-left-color',
+    'border-left-style',
+    'border-left-width',
+    'border-right',
+    'border-right-color',
+    'border-right-style',
+    'border-right-width',
+    'border-top',
+    'border-top-color',
+    'border-top-style',
+    'border-top-width',
+  ],
+  'border-block': [
+    'border-block-start',
+    'border-block-start-color',
+    'border-block-start-style',
+    'border-block-start-width',
+    'border-block-end',
+    'border-block-end-color',
+    'border-block-end-style',
+    'border-block-end-width',
+    'border-top-color',
+    'border-top-style',
+    'border-top-width',
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+  ],
+  'border-block-end': [
+    'border-block-end-color',
+    'border-block-end-style',
+    'border-block-end-width',
+    'border-top-color',
+    'border-top-style',
+    'border-top-width',
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+  ],
+  'border-block-start': [
+    'border-block-start-color',
+    'border-block-start-style',
+    'border-block-start-width',
+    'border-top-color',
+    'border-top-style',
+    'border-top-width',
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+  ],
+  'border-bottom': [
+    'border-bottom-color',
+    'border-bottom-style',
+    'border-bottom-width',
+    'border-block-start-color',
+    'border-block-start-style',
+    'border-block-start-width',
+    'border-block-end-color',
+    'border-block-end-style',
+    'border-block-end-width',
+  ],
+  'border-color': [
+    'border-block-color',
+    'border-block-start-color',
+    'border-block-end-color',
+    'border-inline-color',
+    'border-inline-start-color',
+    'border-inline-end-color',
+    'border-bottom-color',
+    'border-left-color',
+    'border-right-color',
+    'border-top-color',
+  ],
+  'border-image': [
+    'border-image-outset',
+    'border-image-repeat',
+    'border-image-slice',
+    'border-image-source',
+    'border-image-width',
+  ],
+  'border-inline': [
```

Comment:
Example: Should this sort _above_ `border-right` entirely, not just the children?